docker-assets/manageiq.conf

```diff
@@ -67,8 +67,8 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   RewriteCond %{REQUEST_URI} ^/ansibleapi
   RewriteCond %{REQUEST_URI} !(.*)/$
   RewriteRule ^(.*)$ $1/ [R,NE,L]
-  ProxyPass /ansibleapi/ http://localhost:54321/api/
-  ProxyPassReverse /ansibleapi/ http://localhost:54321/api/
+  ProxyPass /ansibleapi/ http://#{ANSIBLE_SERVICE_NAME}/api/
```

Comment:
Also, I don't think we route from the apache pod to ansible in containers (at least not on purpose). We hit the ansible service directly. Are we expecting to expose that service to people who are not our servers?